grafast/grafserv/src/servers/deno.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  convertHandlerResultToResult,
+  GrafservBase,
+  normalizeRequest,
+} from "grafserv";
+
+import type {
+  GrafservBodyJSON,
+  GrafservConfig,
+  RequestDigest,
+  Result,
+} from "grafserv";
+
+declare global {
+  namespace Grafast {
+    interface RequestContext {
+      ctx: Deno.ServeHandlerInfo;
```

Comment:
This needs to be namespaced to Deno:

```suggestion
      denov1: {
        ctx: Deno.ServeHandlerInfo;
      };
```